fibers/io-wakeup.scm

```diff
@@ -110,11 +110,11 @@ of the operation."
 
 (define (wait-until-port-readable-operation port)
   "Make an operation that will succeed when PORT is readable."
-  (make-read-operation (try-ready readable? port) port))
+  (make-read-operation (try-ready readable? port port-read-wait-fd) port))
 
 (define (wait-until-port-writable-operation port)
   "Make an operation that will succeed when PORT is writable."
-  (make-write-operation (try-ready readable? port) port))
+  (make-write-operation (try-ready writable? port port-write-wait-fd) port))
```

Comment:
Maybe this should be fixed in a separate commit, but the use of `readable?` here in `wait-until-port-writable-operation` looks like a bug.

Reply:
Looks like a bug to me too.

Reply:
Done in https://github.com/wingo/fibers/pull/98